lfs/transfer_queue.go

```diff
@@ -231,7 +276,12 @@ func (q *TransferQueue) transferWorker() {
 		}
 
 		if err := transfer.Transfer(cb); err != nil {
-			q.errorc <- err
+			if q.canRetry(err) {
```

Comment:
One of them goes away when the old api support is removed, an extraction for the remaining two is probably overkill.